src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationKeyVaultOptions.cs

```diff
@@ -52,5 +56,24 @@ public AzureAppConfigurationKeyVaultOptions SetSecretResolver(Func<Uri, ValueTas
             SecretResolver = secretResolver;
             return this;
         }
+
+        /// <summary>
+        /// Sets the refresh interval for periodically reloading a secret from Key Vault. Refresh interval must be greater than 1 hour. Default refresh interval is 12 hours.
+        /// Any refresh operation triggered using <see cref="IConfigurationRefresher"/> will not update the value for a Key Vault secret until the cached value for that secret has expired.
+        /// </summary>
+        /// <param name="key">Key of the Key Vault reference in Azure App Configuration.</param>
+        /// <param name="refreshInterval">Minimum time that must elapse before the secret is reloaded from Key Vault.</param>
+        public AzureAppConfigurationKeyVaultOptions SetSecretRefreshInterval(string key, TimeSpan? refreshInterval = null)
```

Comment:
Perhaps the name 'key' doesn't explain what is for. Can we have it a bit more specific? i.e. 'secretReferenceKey'

Reply:
Done.